cts/cli/regression.tools.exp

```diff
@@ -23,6 +23,433 @@ A new shadow instance was created. To begin using it, enter the following into y
 </cib>
 =#=#=#= End test: Validate CIB - OK (0) =#=#=#=
 * Passed: cibadmin       - Validate CIB
+=#=#=#= Begin test: List all available cluster options =#=#=#=
+Pacemaker cluster options
+
+Also known as properties, these are options that affect behavior across the entire cluster. They are configured within cluster_property_set elements inside the crm_config subsection of the CIB configuration section.
+
+  * dc-version: Pacemaker version on cluster node elected Designated Controller (DC)
+    * Includes a hash which identifies the exact revision the code was built from. Used for diagnostic purposes.
+    * Allowed values: string (default: )
+  * cluster-infrastructure: The messaging layer on which Pacemaker is currently running
+    * Used for informational and diagnostic purposes.
+    * Allowed values: string (default: )
+  * cluster-name: An arbitrary name for the cluster
+    * This optional value is mostly for users' convenience as desired in administration, but may also be used in Pacemaker configuration rules via the #cluster-name node attribute, and by higher-level tools and resource agents.
+    * Allowed values: string (default: )
```

Comment:
Either way would be fine. I lean slightly toward blank lines, mostly because I like the idea of "Allowed values" being separated to make it easier to see at a glance.

pcs already does terminal-based line wrapping FWIW, though it will be pulling from our XML output first. It wouldn't hurt for us to do line wrapping for text output as a wishlist item. Mostly noting that users of high-level tools may already have that.

Reply:
Blank line would be fine